providers/kit/doc.go

```diff
@@ -3,6 +3,7 @@
 
 /*
 `kit` provides a small adapter required to use go-kit/log in logging gRPC middlewares.
- Please see examples for examples of use.
+
+	Please see examples for examples of use.
```

Comment:
I think this and the other three instances of this change is wrong, can you revert them, please?